File: articles/large-language-models-and-procedural-reasoning/java/LLMStatus.java
public class LLMStatus {
  static String classify(double score, String stakes) {
    if (stakes.equals("high") && score < 1.0) return "escalate";
    if (score >= 0.8) return "pass";
    return "review";
  }
  public static void main(String[] args) { System.out.println(classify(0.67, "medium")); }
}

File: articles/large-language-models-and-procedural-reasoning/rust/llm_status.rs
fn classify(score: f64, stakes: &str) -> &'static str {
    if stakes == "high" && score < 1.0 { "escalate" }
    else if score >= 0.8 { "pass" }
    else { "review" }
}
fn main() { println!("{}", classify(0.67, "medium")); }
